Podcast/UserSeriesSupplierView.swift

```diff
@@ -89,42 +93,27 @@ class UserSeriesSupplierView: UIView {
             make.leading.equalToSuperview()
             make.height.equalTo(lineseparatorHeight)
         }
-        
-//        if let users = supplier as? [User] {
-//            setupWithUsers(users: users, feedContext: feedContext)
-//        }
-//
-//        if let series = supplier as? Series {
-//            setupWithSeries(series: series)
-//        }
     }
     
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
     
-    func prepareForReuse() {
-        for view in contextImages.subviews {
-            view.removeFromSuperview()
-        }
-    }
-    
     func setupWithUsers(users: [User], feedContext: FeedContext) {
-        contextImages.arrangedSubviews.forEach { $0.removeFromSuperview() }
+//        contextImages.arrangedSubviews.forEach { $0.removeFromSuperview() }
 
         if users != [] {
-            contextLabel.text = ""
+            var contextString = ""
+
             users.enumerated().forEach { (i,user) in
-                if i >= 3 {
-                    return
-                }
-                contextLabel.text = contextLabel.text! + user.fullName()
+                guard let imageView = contextImages.arrangedSubviews.first as? ImageView, i < 3 else { return }
```

Comment:
let's comment this indicating that right now our contextImages stackview only support one user image